az-core/src/test/java/azkaban/utils/PropsTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014 LinkedIn Corp.
```

Comment:
See the dev guide for how to update the copyright automatically.

Reply:
Copied the unit test template from another file, will modify that.